mapr/README.md

```diff
@@ -12,16 +12,41 @@ Follow the instructions below to install and configure this check for an Agent r
-4. Add `/opt/mapr/lib/` to your `/etc/ld.so.conf` (or a file in `/etc/ld.so.conf.d/`). This is required to help the *mapr-streams-library* used by the Agent to find the MapR shared libraries.
-5. Generate a [long-lived ticket][8] for the `dd-agent` user.
-6. Make sure the ticket is readable by the `dd-agent` user.
-7. Configure the integration (see below).
+Prerequisites:
+- [MapR Monitoring][16] needs to be running correctly.
+- You need an available [MapR user][10] (same name, password, uid and gid) with the 'consume' permission on the `/var/mapr/mapr.monitoring/metricstreams` stream. This may be an already existing user or a newly created user. If you want the user user to be called `dd-agent`, you have to create it before installing the agent.
+- You need to generate a [service long-lived ticket][8] for this user that is readable by the `dd-agent` user.
 
-**Note**: If you don't have "security" enabled in the cluster, you can continue without a ticket.
 
+Installation steps for each node:
+1. [Install][2] the Agent
+2. Install the library *mapr-streams-library* with the following command: `sudo -u dd-agent /opt/datadog-agent/embedded/bin/pip install --global-option=build_ext --global-option="--library-dirs=/opt/mapr/lib" --global-option="--include-dirs=/opt/mapr/include/" mapr-streams-python`. If you use Python 3 with Agent 6, replace `pip` by `pip3`.
+3. Add `/opt/mapr/lib/` to your `/etc/ld.so.conf` (or a file in `/etc/ld.so.conf.d/`). This is required for the *mapr-streams-library* used by the Agent to find the MapR shared libraries.
+4. Reload the libraries by running `sudo ldconfig`
+5. Configure the integration by specifying the ticket location.
+
+
+### Additional notes
+
+- If you don't have "security" enabled in the cluster, you can continue without a ticket.
+
+- If your production environment does not allow compilation tools like gcc (required to build the mapr-streams-library), it is possible to generate a compiled wheel of the library on a development instance and distribute the compiled wheel to the production hosts. The development and production hosts have to be similar enough for the compiled wheel to be compatible.
+You can run `sudo -u dd-agent /opt/datadog-agent/embedded/bin/pip wheel --global-option=build_ext --global-option="--library-dirs=/opt/mapr/lib" --global-option="--include-dirs=/opt/mapr/include/" mapr-streams-python` to create the wheel file on the development machine.
+And then `sudo -u dd-agent /opt/datadog-agent/embedded/bin/pip install <THE_WHEEL_FILE>` on the production machine.
+
+- If you use python3 with Agent6, make sure to replace `pip` by `pip3` when installing the *mapr-streams-library*
```

Comment:
Wouldnt it be better to point to the embeeded python? (`/path/to/python -m pip` )

Reply:
How you would phrase that, copy the big command and replace pip for allowing copy pasting? `sudo -u dd-agent /opt/datadog-agent/embedded/bin/pip3 wheel --global-option=build_ext --global-option="--library-dirs=/opt/mapr/lib" --global-option="--include-dirs=/opt/mapr/include/" mapr-streams-python`

I didn't want to add to much weight to that simple sentence.